content/app/guides/multi-app-solution/preparations/_index.en.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Preparations before making a multi-app solution in Altinn
+linktitle: Preparations
+description: What preparations that should be done before creating a multi-app solution
+weight: 10
+toc: true
+aliases:
+
+- /app/multi-app-solution/preparations/
+
+---
+
+A crucial part of the multi-app solution is to make sure the
+applications are able, and allowed, to communicate. This is
+essential due to the main concept of this solution - the
+instantiation of the receiving application. As a part of the
+process of the original form you will have logic that
+creates a request to the receiving application that starts a
+new instance. This request will go to Altinn Storage in
+order to create and persist the instance object, but the
+request will have credentials from the private user who
+logged in to the original form, thus is not allowed to start
+a new instance owned by the organisation that owns the
+receiving application. As a way to bypass this obstacle, we
+can use a Maskinporten integration to authenticate the
+request on behalf of the organisation owning the receiving
+application. In order to achieve this we need to;
+
+1. Create the integration
+   at [Samarbeidsportalen](https://samarbeid.digdir.no/)
+2. Store the keys from the integration in Azure Keyvault for
+   the organisation
+3. Set up the application to use Azure Keyvault and the
+   client to use Maskinporten
+
+## Maskinporten Integration
+
```

Comment:
if key vault should be handled before this, maybe add it as a prerequisite as well. Feel free to link to the info about requesting KV access as well.